client/package.json

```diff
@@ -40,6 +40,7 @@
     "jest-circus": "26.6.0",
     "jest-resolve": "26.6.0",
     "jest-watch-typeahead": "0.6.1",
+    "lodash.debounce": "^4.0.8",
```

Comment:
This is fine, but if we need more lodash functions later, there's no harm in adding the whole lodash library to the project and using tree shaking to only include the functions we need from the lodash library.
https://webpack.js.org/guides/tree-shaking/
https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/